polaris/polaris/helpers.py

```diff
@@ -28,10 +28,22 @@ def calc_fee(asset: Asset, operation: str, amount: float) -> float:
     return fee_fixed + (fee_percent / 100.0) * amount
 
 
-def render_error_response(description: str) -> Response:
-    """Renders an error response in Django."""
-    data = {"error": description}
-    return Response(data, status=status.HTTP_400_BAD_REQUEST)
+def render_error_response(description: str,
+                          status_code: int = status.HTTP_400_BAD_REQUEST,
+                          content_type: str = "application/json") -> Response:
+    """
+    Renders an error response in Django.
+
+    Currently supports HTML or JSON responses.
+    """
+    resp_data = {
+        "data": {"error": description, "status_code": status_code},
+        "status": status_code,
+        "content_type": content_type
+    }
+    if content_type == "text/html":
```

Comment:
Should this function take the request, and determine the content type based on the header of the request?  That way applications can say whether they want html or json instead of us choosing based on endpoint

Reply:
From a code perspective I see why we wouldn't want to have to add `content_type="text/html"` for each of these calls, but I don't think allowing the caller to decide what content type to receive is appropriate here.

To me, it doesn't make sense for our endpoints that return JSON to optionally return HTML (what would it be?), and in the same vein, our `/interactive_*` endpoints must return an interactive form, not JSON. So since having multiple content types per endpoint doesn't really serve our use cases, I'd vote to keep the approach we have in this branch currently.

An alternative solution would be to create a new function for HTML-returning error responses, instead of using the same function for JSON and HTML errors.